src/constants/plugins/buyPluginList.json

```diff
@@ -309,64 +309,49 @@
     "deepPath": "/"
   },
   {
-    "id": "banxa.bank",
-    "pluginId": "banxa",
-    "paymentTypes": ["bank"],
-    "description": "Fee: 5.5% / Settlement: 5 min - 24 hours\nLimit: $50000/day",
-    "title": "POLi Bank Transfer",
-    "partnerIconPath": "banxa.png",
-    "forCountries": ["GB", "AU", "CA", "NL", "TR"],
-    "cryptoCodes": ["BTC", "LTC", "ETH", "USDT"],
-    "paymentTypeLogoKey": "poli",
-    "deepQuery": {
-      "country": "AU",
-      "payment_id": "5021"
-    }
+    "id": "interac",
+    "pluginId": "interac",
+    "paymentType": "interac",
+    "paymentTypes": ["interac"],
+    "title": "Interac e-Transfer",
+    "description": "Fee: 4-5%\nSettlement: 1 - 2 days",
+    "forCountries": ["CA"],
+    "cryptoCodes": [],
+    "paymentTypeLogoKey": "interac"
   },
   {
-    "id": "banxa.payid",
-    "pluginId": "banxa",
-    "paymentTypes": ["bank"],
-    "description": "Fee: 5.5%\nSettlement: Instant",
+    "id": "payid",
+    "pluginId": "payid",
+    "paymentType": "payid",
+    "paymentTypes": ["payid"],
     "title": "PayID",
-    "partnerIconPath": "banxa.png",
+    "description": "Fee: 4-5%\nSettlement: Instant",
     "forCountries": ["AU"],
-    "cryptoCodes": ["BTC", "LTC", "ETH", "USDT"],
-    "paymentTypeLogoKey": "payid",
-    "deepQuery": {
-      "country": "AU",
-      "payment_id": "5035"
-    }
+    "cryptoCodes": [],
+    "paymentTypeLogoKey": "payid"
   },
   {
-    "id": "banxa.auspost",
-    "pluginId": "banxa",
-    "paymentTypes": ["cash"],
-    "description": "Fee: 5.5%\nSettlement: Instant",
-    "title": "Australia Post",
-    "partnerIconPath": "banxa.png",
-    "forCountries": ["AU"],
-    "cryptoCodes": ["BTC", "LTC", "ETH", "USDT"],
-    "paymentTypeLogoKey": "auspost",
-    "deepQuery": {
-      "country": "AU",
-      "payment_id": "5033"
-    }
+    "id": "pix",
+    "pluginId": "pix",
+    "paymentType": "pix",
+    "paymentTypes": ["pix"],
+    "description": "Fee: 3-4%\nSettlement: 5 min - 24 hours",
+    "title": "Pix",
+    "// Do not enable until we have a Pix logo": "",
```

Comment:
Still enabled

Reply:
Didn't want to block this merge for 3.19 and we can easily patch later. I created a task for design to create one.